### ISSUE TITLE
Improving accuracy

### ISSUE THREAD

**Issue by Mustaphajudi** [OpenMOSS/MOSS-Transcribe-Diarize#3]
Is there fine tune script to enhance accuracy of specific language or dialect ?

**Comment by zimoCCC** [comment]
We are preparing to open-source the fine-tuning code.

**Comment by trouble2maker2** [comment]
> We are preparing to open-source the fine-tuning code.

Thank you for the great work! I noticed your comment about preparing to open-source the fine‑tuning code. Could you please share any planned release date or rough timeline? We are very interested in using it to enhance accuracy for specific languages/dialects.

**Comment by zimoCCC** [comment]
> > We are preparing to open-source the fine-tuning code.
> 
> Thank you for the great work! I noticed your comment about preparing to open-source the fine‑tuning code. Could you please share any planned release date or rough timeline? We are very interested in using it to enhance accuracy for specific languages/dialects.

Coming very soon.

**Comment by zimoCCC** [comment]
Thank you for your patience. We have released the fine-tuning code and documentation. Please see FINETUNING.md for the data format and training instructions.
